src/SALib/analyze/discrepancy.py

```diff
@@ -0,0 +1,152 @@
+from typing import Dict
+
+import numpy as np
+from scipy.stats import qmc
+
+from SALib.analyze import common_args
+from SALib.util import read_param_file, ResultDict, _check_groups
+
+
+def analyze(
+    problem: Dict,
+    X: np.ndarray,
+    Y: np.ndarray,
+    method: str = "WD",
+    print_to_console: bool = False,
+    seed: int = None,
+):
+    """Discrepancy indices.
+
+    Parameters
+    ----------
+    problem : dict
+        The problem definition
+    X : numpy.ndarray
+        A NumPy array containing the model inputs
+    Y : numpy.ndarray
+        A NumPy array containing the model outputs
+    method : str
+        Type of discrepancy, can be 'CD', 'WD', 'MD' or 'L2-star'.
+        Refer to `scipy.stats.qmc.discrepancy` for more details. Default is WD.
+    print_to_console : bool
+        Print results directly to console (default False)
+    seed : int
+        Seed value to ensure deterministic results
+        Unused, but defined to maintain compatibility.
+
+    Notes
+    -----
+    Compatible with:
+        all samplers
+
+    Based on 2D sub projections of ``[Xi,Y]``, the discrepancy of each sample
+    is calculated which gives a value for all ``Xi``. This information is used
+    as a sensitivity indice.
```

Comment:
```suggestion
    as a sensitivity index.
```

Or a slightly more verbose option:


```
    as a measure of sensitivity.
```